_data/simple-icons.json

```
@@ -4852,6 +4852,11 @@
                 ]
             }
         },
+        {
+            "title": "Gsmarena",
+            "hex": "F3F3F3",
+            "source": "https://www.gsmarena.com/"
```

The website itself is consistently referred to as "GSMArena.com"; it seems to be only their YouTube channel that's referred to as "GSMArena".
```suggestion
            "source": "https://www.gsmarena.com/",
            "aliases": {
                "aka": [
                    "GSMArena.com"
                ]
            }
```

---

We might get away with just calling the icon GSMArena.com then. Won't need the alias if the full name is part of the main title!

---

Agreed; that would be better 👍🏻